internal/storage-service/service.go

```diff
@@ -18,13 +18,20 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"karavi-authorization/internal/types"
+	storage "karavi-authorization/cmd/karavictl/cmd"
 	"karavi-authorization/pb"
+	"net/url"
 	"strings"
 
+	"github.com/dell/goscaleio"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// KbInGb is the number if kilobytes in a gigabyte. Used for converting Powerflex volume size in Kb to Gb
```

Comment:
minor: "is the number of" instead of "is the number if"

Reply:
Fixed.